src/edu/psu/sweng500/team8/play/GameSession.java

```diff
@@ -106,12 +123,23 @@ public void doRedo() {
 	public void enterNumber(CellCoordinates cellCoordinates, int number) {
 		
 		Cell currentCell = board.getCell(cellCoordinates.getRowIndex(), cellCoordinates.getColumnIndex());
-		SudokuAction sudokuAction = new SudokuAction(cellCoordinates, currentCell.getNumber(), number);
 		
-		/* TODO Actual Board Update to be implemented */
+		/* TODO - come back after PencilMarkMaanger is implement

```

Comment:
where is PencilMarkManager implementation. I merged all the code, but all I see is 

throw new UnsupportedOperationException();